tests/utils_tests/test_http.py

```diff
@@ -288,6 +288,19 @@ def test_parsing_rfc850(self):
         parsed = parse_http_date('Sunday, 06-Nov-94 08:49:37 GMT')
         self.assertEqual(datetime.utcfromtimestamp(parsed), datetime(1994, 11, 6, 8, 49, 37))
 
+    def test_parsing_rfc850_year_50(self):
+        year = datetime.utcnow().year % 100
```

Comment:
This comment from the [earlier PR](https://github.com/django/django/pull/10749/files#r244189594) still applies:

> The test should used a fixed year and find a way to `mock parse_http_date`'s way of obtaining a current year.

Reply:
Let me know if you need assistance with that.

You should be able to achieve it by using a `mock('django.utils.http.datetime.datetime.utcnow')` context manager with a `return_value=datetime(...)`.